tests/integration/v4/provider_charm/charmcraft.yaml

```diff
@@ -26,3 +26,7 @@ parts:
       - libssl-dev
       - pkg-config
       - rustc
+
+charm-libs:
+  - lib: tls-certificates-interface.tls_certificates
+    version: "4"
```

Comment:
Sweet, just learned about this, it would probably be good to use that instead of vendoring in our charms.

Reply:
I didn't know either! This feature should also allow us leverage Dependabot to bump our lib, assuming all we need is to change the string with the version number.

Reply:
Wow I didn't even make that connection!